### ISSUE TITLE
HTTPS for qgroundcontrol.com

### ISSUE THREAD

**Issue by jakehoward** [mavlink/qgroundcontrol#8998]
Tell us a bit about the feature:
- What problem does it solve?

I think it's important for all sites to use https nowadays, but especially those that lead you towards downloading software that you will run on your computer. The specific problem that's most worrying is that someone could trivially hijack the connection and swap the download link for a link to a malicious site. Some of the secondary issues are:
- privacy
- information integrity (I want to be confident any advice, instructions, etc I read came from the qgroundcontrol team)
- confidence in the brand/team/tech

- What does it do?
1. Serve qgroundcontrol.com on https
1. add a redirect from http -> https
- What flight stacks must it work with? (All, PX4, ArduPilot)
n/a
- Relevant vehicle types? (All, multirotor, fixed-wing, VTOL, submarine, etc.)
n/a
- Are there other systems that have this feature?
n/a
- What communication/integration standards does the feature rely on (e.g. MAVLink commands etc.)
n/a

Any additional context you can provide will make the feature easier to evaluate (e.g. mockups, detailed specification, etc.)

Let me know if you would like me to help out with this feature.

**Comment by DonLakeFlyer** [comment]
Makes sense since the downloads come from there. @LorenzMeier Can you deal with this, or find someone to do it?
